FanartHandler/FanartSelectedOther.cs

```diff
@@ -151,6 +151,16 @@ public void RefreshGenericSelectedProperties(Utils.SelectedType property, ref st
             }
             AddSelectedGenreProperty(SelectedGenre, SelectedItem, property);
 
+            // Trakt remove ClearArt for pages
+            if (SelectedItem == "Next Page" ||
+                SelectedItem == "Previous Page" ||
+                SelectedItem.StartsWith("Next ") && SelectedItem.EndsWith(" Days") ||
+                SelectedItem.StartsWith("Previous ") && SelectedItem.EndsWith(" Days"))

```

Comment:
This is not a very good option, it’s better not to get attached to the names, otherwise we may get a false positive. Since we know for sure that we are working with Path, we can rely on the properties of the element. `IsPrevPageItem | IsNextPageItem` or `IconImage = "traktPreviousPage.png" | IconImage = "traktNextPage.png"` or `IsFolder = true`
https://github.com/trakt/Trakt-for-Mediaportal/blob/e817f186665663878780bc78dd06d70c56260d10/TraktPlugin/GUI/GUIAnticipatedMovies.cs#L422

Reply:
Great idea! I agree that it was not a good option but did not know what I could use. I have changed it to IsFolder.